models/shopify_graphql.yml

```diff
@@ -0,0 +1,1202 @@
+      - name: quantity_refunded
+        description: The quantity of goods ORDERED on this day that have been refunded. 
+      - name: quantity_net
+        description: Net quantity sold from this shop on this day. Excludes refunds.
+      - name: count_variants_sold
+        description: Distinct product variants sold on this day (includes refunds).
+      - name: count_products_sold
+        description: Distinct products sold on this day (includes refunds).
+      - name: quantity_gift_cards_sold
+        description: Quantity of gift cards sold on this day.
+      - name: quantity_requiring_shipping
+        description: Quantity of goods sold on this day that require shipping.
+      - name: count_abandoned_checkouts
+        description: Count of abandoned checkouts on this day.
+      - name: count_customers_abandoned_checkout
+        description: Count of distinct customers who abandoned checkouts on this day.
+      - name: count_customer_emails_abandoned_checkout
+        description: Count of distinct customer emails who abandoned checkouts on this day.
+      - name: count_fulfillment_attempted_delivery
+        description: Count of distinct fulfillments on this day where the delivery of the shipment was attempted, but unable to be completed.
+      - name: count_fulfillment_delivered
+        description: Count of successful distinct fulfillments successfully delivered on this day.
+      - name: count_fulfillment_failure
+        description: Count of failed distinct fulfillments on this day. Something went wrong when pulling tracking information for the shipment, such as the tracking number was invalid or the shipment was canceled.
+      - name: count_fulfillment_in_transit
+        description: Count of distinct fulfillments in transit on this day. Each shipment is being transported between shipping facilities on the way to its destination.
+      - name: count_fulfillment_out_for_delivery
+        description: Count of distinct fulfillments being delivered to their final destination.
+      - name: count_fulfillment_ready_for_pickup
+        description: Count of distinct fulfillments ready for pickup at a shipping depot.
+      - name: count_fulfillment_picked_up
+        description: Count of distinct fulfillments successfully picked up on this day.
+      - name: count_fulfillment_label_printed
+        description: Count of distinct fulfillments for which a purchased shipping label has been printed on this day.
+      - name: count_fulfillment_label_purchased
+        description: Count of distinct fulfillments for which a shipping label has been purchased (but not printed yet) on this day.
+      - name: count_fulfillment_confirmed
+        description: Count of confirmed distinct fulfillments. This is the default status when no other information is available about a fulfillment.
+      - name: count_fulfillment_delayed
+        description: Count of delayed distinct fulfillments.
```

Comment:
None of these fulfillment metrics are included in the end model. Is that expected?

Reply:
They should be included if you set `shopify_gql_using_fulfillment_event: True`